product.php

```diff
@@ -0,0 +1,185 @@
+                <label>Quantity
+                    <select id="quantity" name="quantity" size="1" required>
+                        <option value="1">1</option>
+                        <option value="2">2</option>
+                        <option value="3">3</option>
+                        <option value="4">4</option>
+                        <option value="5">5</option>
+                        <option value="6">6</option>
+                        <option value="7">7</option>
+                        <option value="8">8</option>
+                        <option value="9">9</option>
+                        <option value="10">10</option>
+                        <option value="11">11</option>
+                        <option value="12">12</option>
+                    </select>
+                </label>
+                <img src="https://cdn1.iconfinder.com/data/icons/material-core/20/shopping-cart-128.png" style="width:50px; height:50px" title="Cart Icon" alt="Shopping cart"/>
+                <input type="submit" value="Add to Cart">
+            </form>
+
+            <br>
+            <!--social media links-->
+            <img src="https://cdn4.iconfinder.com/data/icons/43-social-media-line-icons/24/Share-128.png" style="width:50px; height:50px" title="Share" alt="Share Icon"/>
+            <a href="https://twitter.com/?lang=en">
+                <img src="https://cdn3.iconfinder.com/data/icons/peelicons-vol-1/50/Twitter-128.png" style="width:50px; height:50px" title="Twitter" alt="Twitter Icon"/>
+            </a>
+            <a href="https://www.facebook.com/">
+                <img src="https://cdn3.iconfinder.com/data/icons/peelicons-vol-1/50/Facebook-128.png" style="width:50px; height:50px" title="Facebook" alt="Facebook Icon"/>
+            </a>
+            <a href="https://www.instagram.com/">
+                <img src="https://cdn3.iconfinder.com/data/icons/picons-social/57/38-instagram-128.png" style="width:50px; height:50px" title="Instagram" alt="Instagram Icon"/>
+            </a>
+
+        </div>
+    </div>
+
+    <!--Thumbnail Images-->
+    <div class="row"> <!--four-->
+        <div class="six columns">
+            <a href="">
+                <img src="https://images.pexels.com/photos/913136/pexels-photo-913136.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940" style="width:82px; height:86px" title="Main Image 1" alt="Dessert"/>
+            </a>
+            <a href="">
+                <img src="https://images.pexels.com/photos/913136/pexels-photo-913136.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940" style="width:82px; height:86px" title="Main Image 2" alt="Dessert"/>
+            </a>
+            <a href="">
+                <img src="https://images.pexels.com/photos/913136/pexels-photo-913136.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940" style="width:82px; height:86px" title="Main Image 3" alt="Dessert"/>
+            </a>
+        </div>
+
+        <!--BLANK SIX CLOUMNS-->
```

Comment:
Please remove these comments as well as the number ones like `<!--four-->` in this file. As much as possible we want to keep our code free of comments unless they are describing something important. `<!--Product Description-->` is good because it lets all developers know where the product description begins